### PR TITLE
(validator.js) Add missing options to isInt method

### PR DESCRIPTION
**Pull request by kvcpr**
Missing options:
`allow_leading_zeroes`: https://github.com/chriso/validator.js/blob/master/src/lib/isInt.js#L13
`lt`: https://github.com/chriso/validator.js/blob/master/src/lib/isInt.js#L20
`gt`: https://github.com/chriso/validator.js/blob/master/src/lib/isInt.js#L21

Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/chriso/validator.js/blob/master/src/lib/isInt.js
- [x] Increase the version number in the header if appropriate.

@tgfjt @chrootsu @IOAyman @louy

### CODE REVIEW

File: validator/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for validator.js v5.7.0
```
Comment (tgfjt) [inline validator/index.d.ts]: It seems to be 6.2.1? 👍

Comment (kacepe) [inline validator/index.d.ts]: I think yes, but I didn't make more changes to increase version number. I want to do it in the evening.

File: validator/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for validator.js v5.7.0
+// Type definitions for validator.js v6.2.1
```
Comment (DanielRosenwasser) [inline validator/index.d.ts]: This version number should only be `major.minor` - can you add a tslint.json with the following content?

```json
{ "extends": "../tslint.json" }
```

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: validator/index.d.ts (hunk 2 above)
Comment (kacepe) [inline validator/index.d.ts]: Sure, done.

Comment (kacepe) [inline validator/index.d.ts]: It isn't pass a test with this.

Comment (DanielRosenwasser) [inline validator/index.d.ts]: Weird (@mhegazy?) - anyway, can you just switch this to `v6.2`? After that we can pull this in.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: validator/index.d.ts (hunk 2 above)
Comment (kacepe) [inline validator/index.d.ts]: @DanielRosenwasser done

**Pull request merged**